README.md

```diff
@@ -27,21 +27,29 @@ php composer.phar update
 
 To use the AWS Service Provider, you must register the provider when bootstrapping your Laravel application.
 
-Publish the package configuration using Artisan.
+By default, the package uses the following environment variables to auto-configure the plugin without modification:
+```
+AWS_KEY
```

Comment:
We use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY ` in the SDK, so it would make sense to standardize on that, right?